crates/pop-cli/src/commands/mod.rs

```diff
@@ -33,9 +33,9 @@ pub(crate) enum Command {
 	#[clap(alias = "u", about = about_up())]
 	#[cfg(any(feature = "parachain", feature = "contract"))]
 	Up(up::UpArgs),
-	/// Test a smart contract.
+	/// Test a project.
```

Comment:
```suggestion
	/// Test a Rust project.
```
Maybe more specific?

Reply:
Agreed! https://github.com/r0gue-io/pop-cli/pull/466/commits/a2b8270dbf54db7bef23d203aaf6488216e705d0